### PR TITLE
Changed it to a simple weather widget

### PR DESCRIPTION
**Pull request by damiththa**
Made changes now so it now uses css formatting and also utilizing other info from Forecast.io API call including _icon_ for nice animation.

### CODE REVIEW

**Comment by damiththa** [comment]
Here is what it'll look like, where weather _icon_ is an animation that uses `html5` `canvas` tag.

forecastio_facelift

File: weather.cfm [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Weather forecast widget</title>   
+    <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/picnic/6.1.1/picnic.min.css">
+    <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/pure/0.6.0/pure-min.css">
+    <link rel="stylesheet" type="text/css" href="Styles/weather.css">
+</head>
+<body>
+<!--Components-->
+<cfset weather = createObject("component", "ForecastCFML.Components.weather") />
```
Comment (damiththa) [inline weather.cfm:14]: Make sure path to the component is mapped correctly

File: Components/weather.cfc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<cfcomponent output="no">
+  <cfset config=createObject("component", "ForecastCFML.Components.weather") />
```
Comment (damiththa) [inline Components/weather.cfc:2]: Make sure path to the component is mapped correctly

**Comment by senf666** [comment]
Thanking you!

**Comment by damiththa** [comment]
Thank you and you are welcome.

**Comment by brianmweis** [comment]
Anyway to update for darksky. I tried uploading and it did not work. I replaced the app secret/api key and changed url but did not work.

**Comment by brianmweis** [comment]
Anyway to update for darksky. I beleive they took this over.

**Pull request merged**